android/src/com/mapswithme/maps/review/Review.java

```diff
@@ -7,54 +7,42 @@
 
 public class Review implements Parcelable
 {
-  @Nullable
-  private final String mReview;
-  @Nullable
-  private final String mReviewPositive;
-  @Nullable
-  private final String mReviewNegative;
+  private final long mDate;
+  private final float mRating;
   @NonNull
   private final String mAuthor;
-  @NonNull
-  private final String mAuthorAvatar;
-  private final float mRating;
-  private final long mDate;
+  @Nullable
+  private final String mPros;
```

Comment:
Why have you changed positive/negative reviews by pros and cons? What's wrong with old names?